tests/fixtures/site.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+
+@pytest.fixture
+def site_matrix(factories):
+    projects = factories['project'](batch=3)
+    locales = factories['locale'](batch=3)
+    pl_kwa = []
+    resource_kwa = []
+    tr_kwa = []
```

Comment:
I assume `kwa` stands for `kwargs`? And I'd rather have long and clear variable names than acronyms, for the sake of people who will read this later and will have to figure out / remember what `pl` and `tr` stand for. 

As a side note, Elon Musk hates acronyms, too: https://pbs.twimg.com/media/CF4cFtLUgAASqYO.png :P

Reply:
these are local variables in a test, surely we can be a little bit relaxed here